web-ui/src/main/resources/catalog/js/admin/SystemSettingsController.js

```diff
@@ -41,6 +41,24 @@
       return filtered;
     }
   });
+
+  /**
+   * Filters internal settings used by GeoNetwork,
+   * not intended to be configured by the user.
+   */
+  module.filter('hideGeoNetworkInternalSettings', function() {
+    return function(input) {
+      var filtered = [];
+      var internal = ['system/userFeedback/lastNotificationDate'];
```

Comment:
As a future improvement, if a setting is internal or not can be defined in the database for each setting, instead of in the Angular JS controller.

Reply:
Right, actually there's a field `internal` in the database that is used to ignore them in the retrieve settings service used for the public UI, for example for passwords for mail server, etc. that should not be returned.

But not the same as this one. Not sure if are any other settings like this one `system/userFeedback/lastNotificationDate`, but a field in the database to track them seem an improvement.